src/main/java/com/atlassian/db/replica/internal/aurora/AuroraJdbcUrl.java

```diff
@@ -0,0 +1,25 @@
+package com.atlassian.db.replica.internal.aurora;
+
+public class AuroraJdbcUrl {
```

Comment:
It's internal but it's a parameter for the API constructor (AuroraCluster). Please make it API or fix the constructor to use API only classes.